package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "npm run format && npm run test:dev",
     "format": "run-s format:*",
-    "format:lint": "eslint --ignore-path .gitignore --fix --cache --format=codeframe --max-warnings=0 \"src/**/*.js\"",
+    "format:lint": "eslint --ignore-path .eslintignore --fix --cache --format=codeframe --max-warnings=0 \"src/**/*.js\"",
```

Comment:
It seems that the reason to change this would be not to lint the new files in `src/node-compat/assets`, is this correct?
If so, I think fixing the linting errors in those files should be the fix instead.

Reply:
those files are going to be injected into the user's bundle.
the lint errors are about the parser that eslint uses, which does not like es modules. so no chance to fix it.

see https://github.com/netlify/netlify-plugin-edge-handlers/runs/1210596968#step:5:32

Reply:
We should fix `.eslintrc.json` instead to allow linting both ES modules and CommonJS, e.g. using `overrides`.